.github/scripts/check-for-update.sh

```diff
@@ -57,7 +57,7 @@ while [ $i -le $count ]; do
     echo "checking $item"
 
     case "$item" in
-      *".github"*|*"experimental"*)
+      (*".github"*)
```

Comment:
Are you sure this `(` belongs here if it wasn't there before?

Reply:
Both are valid `sh` (checked with `shellcheck`), I think I should get rid of it though, looked up the case syntax and it doesn't need the first bracket.

Reply:
https://devhints.io/bash (search for case/switch)

Reply:
Weird syntax is weird 🤷